torch_xla/csrc/device.cpp

```diff
@@ -72,6 +77,10 @@ const torch::lazy::BackendDevice* GetDefaultDevice() {
   return &default_device;
 }
 
+torch::lazy::BackendDevice GetVirtualDevice() {
+  return ParseDeviceString("SPMD:0");
```

Comment:
We won't want to use device strings for virtual device. Instead, once virtual device is implemented, this will return a global variable, like `GetCurrentDevice()` below.

No action needed.